test/libsolidity/syntaxTests/license/license_double2.sol

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: GPL-3.0
+// SPDX-License-Identifier: MIT
```

Comment:
I'd add a test with multiple declarations in the same multi-line comment and also another with declarations on the same line.

Reply:
Also another with two identical rather than different licenses (it will still be treated as error, right?)